doc/releases/release-notes-4.0.rst

```diff
@@ -1118,6 +1334,19 @@ Tests and Samples
 * Together with the deprecation of :ref:`native_posix<native_posix>`, many tests which were
   explicitly run in native_posix now run in :ref:`native_sim<native_sim>` instead.
   native_posix as a platform remains tested though.
+* Added documentation (readme) for the erase_blocks flash test
+* Extended the tests of counter_basic_api with a testcase for counters wihtout alarms
+* Excluded NXP RW612 based boards from the WiFi test suite, as these boards require binary blobs
+  be downloaded in order to build as expected
+* Added support for testing SDMMC devices to the fatfs API test
+* Extended net/vlan to add IPv6 prefix config to each vlan-iface
+* Enhanced the camera fixture test by adding a color bar to enable automation
+* Fixed floating point logging issue in the video driver sample code
+* Added a number crunching (maths such as FFT, echo cancellation) sample using optimized an
+  library for the NXP ADSP board
+* Tailored the SPI_LOOPBACK test to the limitations of NXP Kinetis MCU's
+* Enabled the video sample to run video capture (samples/drivers/video)
+* Add arguments to the twister runner for SoC, Platform and Simulation
```

Comment:
as before, don't go to the level of details where you're describing changes or tweaks being made to tests (to put it differently: if the release notes are effectively going to the level of detail were it's one bullet point per PR or commit, then we'd rather point users directly to the GitHub interface to see what's new by themselves :) )